src/Microsoft.OData.Core/Json/JsonReader.cs

```diff
@@ -1491,21 +1518,87 @@ private void TryPopPropertyScope()
         }
 
         /// <summary>
-        /// Skips all whitespace characters in the input.
+        /// Vectorized implementation of whitespace skipping using Vector for cross-platform SIMD.

```

Comment:
Do we have a different implementation of this method that is not vectorized? Why do we have to specify that it's vectorized/SIMD in the method summary? That's an implementation detail.